### DISCUSSION TITLE
Piano Keys

### DISCUSSION THREAD

**Discussion by SeriousM** [monkeytypegame/monkeytype#1759] [Ideas] [answered]
Hi! I wonder if it's possible to use a midi keyboard and a custom layout to play the game with music notes.
This is not a feature request per-se, rather a question if it's possible at all.
I would like to try to implement myself, but before I take a deep dive I just want to check with the experts if it's feasible at all.
Thanks!

**Comment by Miodec** [comment, marked as answer]
Seems kinda possible

https://www.smashingmagazine.com/2018/03/web-midi-api/
